packages/common/src/index.ts

```diff
@@ -154,6 +155,21 @@ export const validateEmail = (email: string): boolean => {
     return re.test(String(email).toLowerCase());
 };
 
+export function validatePassword(password: string): PasswordValidationResult {
```

Comment:
It would be great to have unit tests for this method

Reply:
Done.